core/package.json

```diff
@@ -31,7 +31,7 @@
     "loader/"
   ],
   "dependencies": {
-    "@stencil/core": "^4.12.2",
+    "@stencil/core": "^4.14.0",
```

Comment:
Can we revert this change and `package-lock.json` change? Updating stencil should not be required for this PR.